README.md

```diff
@@ -3,240 +3,90 @@
 [![Build
 status](https://badge.buildkite.com/c2da30c5a1deb1ff6e0ca09c5ec33f7bd0a5b57ea35df4fc15.svg)](https://buildkite.com/granola/mina-indexer)
 
-The Mina indexer ("indexer") is a simplified, and improved version of
-the software collectively called the "[archive
-node](https://github.com/MinaProtocol/mina/tree/develop/src/app/archive)"
-in the Mina codebase.
+The Mina Indexer is a redesigned version of the software collectively
+called the "[Mina archive
+node](https://github.com/MinaProtocol/mina/tree/develop/src/app/archive)."
 
-The indexer replaces the archive node trio of architectural elements
-(Postgres database, Mina daemon, "mina-archiver" process) with a
-system that reconstitutes the network’s historical state solely from
-the precomputed blocks logged from the Mina daemon.
+**Note:** As the project is in active development, be aware that the
+public APIs and functionalities are subject to change.
 
-![High Level Architecture](docs/architecture/indexer_components.png)
+## Motivation
 
-The indexer's primary goals are to be easier to operate and maintain
-while being a superset of the data available in the archive node.
+Building applications with the Mina Archive Node is intricate and
+requires significant time investment. It demands precise configuration
+of a Mina Node, an Archive Node, and a Postgres database. Proficiency
+in SQL and an in-depth understanding of the Mina blockchain are
+crucial for effective operation.
 
-## Warning
+One major challenge with the Archive Node is its reliance on a
+`pg_dump` from an existing node for bootstrapping, ideally within the
+transition frontier. The Mina Indexer addresses this challenge by
+relying on precomputed blocks as the source of truth, eliminating the
+need to rely on someone else's database dump.
 
-The indexer project is in constant development and is in an alpha
-state. Functionality and API definitions will be in flux and are
-subject to change without notice. With that being said, happy hacking!
+Even with proper setup, the system may not capture every block, often
```

Comment:
Because the prior paragraph's final sentence has Mina Indexer as the subject, it seems to me that this paragraph applies to the Indexer (not the archive node).

Reply:
I'll create another PR that makes it more clear